src/Compilers/VisualBasic/Portable/Lowering/ExpressionLambdaRewriter/ExpressionLambdaRewriter.vb

```diff
@@ -47,45 +47,35 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Public ReadOnly Property ElementInitType As NamedTypeSymbol
             Get
-                If _elementInitType Is Nothing Then
-                    _elementInitType = Me._factory.WellKnownType(WellKnownType.System_Linq_Expressions_ElementInit)
-                End If
+                _elementInitType = If(_elementInitType, _factory.WellKnownType(WellKnownType.System_Linq_Expressions_ElementInit))
                 Return _elementInitType
             End Get
         End Property
```

Comment:
The changes to `VisualBasic/Portable/Lowering/*` appear unrelated. Please revert.